pkg/network/protocols/events/consumer_test.go

```diff
@@ -78,6 +78,32 @@ func TestConsumer(t *testing.T) {
 	}
 }
 
+func TestInvalidBatchCountMetric(t *testing.T) {
+	kversion, err := kernel.HostVersion()
+	require.NoError(t, err)
+	if minVersion := kernel.VersionCode(4, 14, 0); kversion < minVersion {
+		t.Skipf("package not supported by kernels < %s", minVersion)
+	}
+
+	program, err := newEBPFProgram(config.New())
+	require.NoError(t, err)
+	defer program.Stop(manager.CleanAll)
+
+	ringBufferHandler := ddebpf.NewRingBufferHandler(1)
```

Comment:
why ringbuffer? what are you doing? why do you override the handler? ringbuffers are supported only from kernel 5.8, so for the naive reader, how is the test not failing on older kernels?

Reply:
I created a function to simplify the logic related to the kernel version. What do you think of this solution?